polaris.shopify.com/pages/[...slug].tsx

```diff
@@ -92,6 +94,7 @@ export const getStaticPaths: GetStaticPaths = async () => {
   const globPath = [
     path.resolve(process.cwd(), 'content/*.md'),
     path.resolve(process.cwd(), 'content/**/*.md'),
+    path.resolve(process.cwd(), 'content/**/**/*.md'),
```

Comment:
This enables us to nest the rule pages in another directory and have the nav look like this 
<img width="287" alt="Screenshot 2023-01-17 at 12 02 33 PM" src="https://user-images.githubusercontent.com/20652326/212963929-0eb44d98-4728-40f1-889d-1e8d84324ad2.png">